cms/templates/js/certificate-details.underscore

```diff
@@ -0,0 +1,40 @@
+<div class="collection-details wrapper-certificate">
+    <header class="collection-header">
+        <h3 class="title">
+            <a href="#" class="toggle detail-toggle <% if (showDetails){ print('hide'); } else { print('show'); } %>-details">
+                <i class="ui-toggle-expansion icon fa fa-caret-<% if (showDetails){ print('down'); } else { print('right'); } %>" aria-hidden="true"></i>
+                <%= name %>
+            </a>
+        </h3>
+    </header>
+
+   <ol class="collection-info certificate-info certificate-info-<% if(showDetails){ print('block'); } else { print('inline'); } %>">
+     <% if (!_.isUndefined(id)) { %>
+         <li class="certificate-id">
+            <span class="certificate-label"><%= gettext('ID') %>: </span>
+            <span class="certificate-value"><%= id %></span>
+        </li>
+     <% } %>
+     <% if (showDetails) { %>
```

Comment:
No need to have certificate descriptions, this list item can be removed. (and potentially the associated field, unless we want to default it to something else or keep it empty.)